src/Services/privateApiService.js

```diff
@@ -1,15 +1,35 @@
 import axios from 'axios';
+import { baseURL } from './Api';
 
 const config = {
     headers: {
-        Group: 01                //Aqui va el ID del equipo!!
+        "Group": 144
     }
 }
 
-const Get = () => {
-    axios.get('https://jsonplaceholder.typicode.com/users', config)
-    .then(res => console.log(res))
-    .catch(err => console.log(err))
+export const getRequest =  async(url,getToken,id = null) =>{
```

Comment:
Me mata el toc, necesito espacios

```suggestion
export const getRequest =  async (url, getToken, id = null) => {
```